### PR TITLE
Add Session Feedback Button

### PR DESCRIPTION
**Pull request by futabooo**
## Issue
- #74 Create Session Feedback page

## Overview (Required)
- Add session feedback button, dummy googleform url

## Links
N/A

## Screenshot
N/A

I tried it. necessary everyone's follow. 🌵 🌵

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/SessionsRepository.java [hunk 1]
```diff
@@ -109,4 +111,8 @@ private void refreshCache(List<Session> sessions) {
     public void setIdDirty(boolean isDirty) {
         this.isDirty = isDirty;
     }
+
+    public Single<Response<Void>> submitSessionFeedback(SessionFeedback sessionFeedback) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/SessionsRepository.java]: I think it's better to create `SessionFeedbackRepository` class which have `submit()` method.

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionFeedbackViewModel.java [hunk 2]
```diff
@@ -60,4 +66,29 @@ public void setSessionTitle(String sessionTitle) {
...
+
+    public void setCallback(@NonNull Callback callback) {
+        this.callback = callback;
+    }
+
+    public interface Callback {
+
+        void onClickSubmitFeedback();
+
+        void onSubmitSuccess();
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionFeedbackViewModel.java]: I think this `onClickSubmitFeedback` and `onSubmitSuccess` methods is not necessary if you call `.subscribe()` in `SessionFeedbackFragment`.

File: app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java [hunk 3]
```diff
@@ -74,4 +93,13 @@ private static Gson createGson() {
         Single<List<Contributor>> getContributors(@Path("owner") String owner,
                 @Path("repo") String repo);
     }
+
+    interface GoogleFormService {
+        // TODO: change test url and add form item
+        @POST("1FAIpQLSeSr-Tn3TnNum2md8l3jCZa0cnMYMXXaouHomubecdaRlGFCQ/formResponse")
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java]: Thanks! Actually we have not created the feedback form yet 🙇

File: app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/SessionsRepository.java (hunk 1 above)
Comment (futabooo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/repository/sessions/SessionsRepository.java]: it is better!
i will

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionFeedbackViewModel.java (hunk 2 above)
Comment (futabooo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionFeedbackViewModel.java]: I will give it a try!

File: app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java (hunk 3 above)
Comment (futabooo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java]: np :)
maybe, new issue create this pr todo

**Comment by konifar** [comment]
Sorry, as I said before, the detailed spec is not decided.
Please wait more 🙇

**Comment by futabooo** [comment]
sorry I rebased it.

After specifying the specifications of feedback items, we will implement the layout and client, so could you this p-r merge?

Before the client conflicts become hard 😭

**Comment by konifar** [comment]
I know what you mean...
Maybe we'll create the feedback page as same as 2016 app.
Okay! I'll merge this first after CI success 😉

**Comment by konifar** [comment]
OMG... Sorry, the conflict occurred. Next time, I'll merge this first. Can you merge this? 🙇

**Comment by futabooo** [comment]
Sure!

**Comment by konifar** [comment]
Thanks for contribution 👍

**Comment by futabooo** [comment]
Thank you for following me!

**Pull request merged**